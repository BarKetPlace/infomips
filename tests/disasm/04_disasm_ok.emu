#-------
#TEST_RETURN_CODE=PASS           
#TEST_COMMENT="Teste fonctionnement standard"
#-------


load ./elf/boucle.o

disasm 0x12
disasm 0x13
disasm 0x0:0x8
disasm 0x0+8
disasm 0x0+0
disasm 0x0+3
disasm 0x0+5

