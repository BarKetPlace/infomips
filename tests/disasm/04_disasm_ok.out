0x14000000	0xfc000000	BNE	I	3	rs rt imm 
0x00000000	0xffffffff	NOP	J	0	
0x08000000	0xfc000000	J	J	1	target 
0x20000000	0xfc000000	ADDI	I	3	rt rs imm 
0x00000020	0xfc0007ff	ADD	R	3	rd rs rt 
0x00000010: 0x00000000 NOP 
0x00000010: 0x00000000 NOP 
0x00000000: 0x2009000c ADDI $9 $0 12 
0x00000004: 0x00005020 ADD $10 $0 $0 
0x00000000: 0x2009000c ADDI $9 $0 12 
0x00000004: 0x00005020 ADD $10 $0 $0 
0x00000000: 0x2009000c ADDI $9 $0 12 
0x00000000: 0x2009000c ADDI $9 $0 12 
0x00000000: 0x2009000c ADDI $9 $0 12 
